File: sdks/node/src/modules/cases.ts

import { Client } from "../client";

export class CasesClient {
  constructor(private client: Client) {}

  async createCase(title: string, description: string) {
    return this.client.request("POST", "/cases", { title, description });
  }

  async updateCase(caseId: string, updates: any) {
    return this.client.request("PUT", `/cases/${caseId}`, updates);
  }
}

File: sdks/python/fiduscan/cases.py

class CasesClient:
    def __init__(self, client):
        self.client = client

    def create_case(self, title, description):
        return self.client.request("POST", "/cases", json={"title": title, "description": description})

    def update_case(self, case_id, updates):
        return self.client.request("PUT", f"/cases/{case_id}", json=updates)
